src/gui/widgets/InstrumentMidiIOView.cpp

```diff
@@ -108,8 +109,6 @@ InstrumentMidiIOView::InstrumentMidiIOView( QWidget* parent ) :
 	midiOutputLayout->addWidget( m_fixedOutputNoteSpinBox );
 	midiOutputLayout->addStretch();
 
-	connect( m_midiOutputGroupBox->ledButton(), SIGNAL( toggled( bool ) ),
-			m_outputChannelSpinBox, SLOT( setEnabled( bool ) ) );
```

Comment:
As above, what if we don't want to enable midi output?

Reply:
Same as above, this makes the midi output channel spin box widget independent from the led button that enables/disables the midi output. That's just so you can more conveniently change the channel without needing to enable the midi output.